frappe/utils/__init__.py

```diff
@@ -833,7 +833,7 @@ def get_site_info():
 	return json.loads(frappe.as_json(site_info))
 
 
-def parse_json(val):
+def parse_json(val: str):
```

Comment:
does accept `str | dict | Any` technically

Reply:
I thought so, too and then made an avoidable logical mistake. That's how I ended up clarifying.

Now, it would have told me: "this ought to be a string, proceed at your own risk."

Reply:
Hmm, yeah, fine if you think about it in that way.